pkg/util/fileutil/file.go

```diff
@@ -181,3 +181,7 @@ func DownloadOrUpdateGitRepo(url string) (path string, err error) {
 
 	return path, nil
 }
+
+func EnvdHomeDir(path ...string) string {
+	return filepath.Join(append([]string{"/home", "envd"}, path...)...)
```

Comment:
if we split `/home` and `envd`, should we also keep `/` isolated like `[]string{"/", "home", "envd"}`?

or if we put `/home` together, maybe it is not necessary to split the `envd`, make it `[]string{"/home/envd"}`?

Reply:
SGTM. Changed to `[]string{"/", "home", "envd"}`